packages/nimble-blazor/Examples/Demo.Shared/Pages/ComponentsDemo.razor

```diff
@@ -218,6 +218,11 @@
                     <NimbleIconKey title="First Name"></NimbleIconKey>
                 </NimbleTableColumnText>
                 <NimbleTableColumnAnchor HrefFieldName="Href" LabelFieldName="LinkLabel" ColumnId="link-column" >Link</NimbleTableColumnAnchor>
+                <NimbleTableColumnEnumText FieldName="StatusCode" KeyType="MappingKeyType.Number" ColumnId="status-column">
```

Comment:
This isn't using the implementation strategy as [outlined in the spec](https://github.com/ni/nimble/blob/main/packages/nimble-components/src/table/specs/table-column-specs/table-column-mapping.md#blazor-integration). Did that not work? Ultimately I think it would be good to not have these string representations of types in the C# API.

Reply:
Thanks for reminding me. I am now using generics.